modules/distribution/src/repository/conf/security/authenticators.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+
+<!--
+  ~ Copyright 2005-2011 WSO2, Inc. (http://wso2.com)
```

Comment:
Recommended fixes can be found in PR[1]

[1]. https://github.com/wso2/product-mdm/pull/166